cranelift/filetests/filetests/isa/x86/bnot-b1.clif

```diff
@@ -0,0 +1,14 @@
+test binemit
+test run
+
+target x86_64
+
+function u0:323() -> b1 system_v {
```

Comment:
```suggestion
function u0:323() -> b1 {
```

Use the default calling convention to fix `test run` on Windows.

Reply:
Ah I missed that, thanks!